sdk/servicebus/azure-servicebus/swagger/generate_attributes_sequence_tuple.py

```diff
@@ -4,10 +4,10 @@
 # --------------------------------------------------------------------------------------------
 
 
-# This is a tool, not a part of the SDK code. Run it with Python 3.6+.
+# This is a tool, not a part of the SDK code. Run it with Python 3.7+.
```

Comment:
As mentioned above, Python 3.6+ includes Python 3.7, so we don't really need to make this change.